.ci/e2e-expected/select-ddl-batching.txt

```diff
@@ -0,0 +1 @@
+ERROR:  INVALID_ARGUMENT: The statement is not suitable for batching. Only DML and DDL statements are allowed for batching.
```

Comment:
I think it will be helpful to add a little more clarity here.

Currently we only support a batch of either DDL or DML statements. List of non-supported statements include (but not. limited to) are SELECT, Connection statement and Instance/Database Admin statements. Other statements are still supported within single execution [feel free to change terminology here.. maybe call it non-batch or serial transactions].

Reply:
it seems we also support a limited format for sql statements.. is tis documented anywhere ?
Can that be included in the error message from either of these:
* while parsing
* while executing ?


 STATEMENT_DELIMITER = ';';
 SINGLE_QUOTE = '\'';

Reply:
I think it would be easier to include this in the README/documentation rather than the error message.  Otherwise the error message will be quite verbose.